lib/ex_doc.ex

```diff
@@ -25,6 +25,7 @@ defmodule ExDoc do
       project: nil,
       retriever: ExDoc.Retriever,
       source_beam: nil,
+      source_ref: "master",
```

Comment:
I don't understand why this is needed. `:source_ref` is not needed in the struct inside ExDoc. We only use it initially when building the source_url.